src/test/java/hw2/ex2/SecondExerciseTest.java

```diff
@@ -0,0 +1,166 @@
+        serviceHeaderLeftPanel.click();
+        WebElement serviceSupportSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Support')]"));
+        WebElement serviceDatesSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Dates')]"));
+        WebElement serviceComplTableSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Complex Table')]"));
+        WebElement serviceSimplTableSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Simple Table')]"));
+        WebElement serviceTableWithPagesSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Table with pages')]"));
+        WebElement serviceDiffElemSubCategoryLeftPanel = driver.findElement(
+                By.xpath("//ul[@class='sub']//li[contains(.,'Different elements')]"));
+        softAssert.assertTrue(serviceSupportSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertTrue(serviceDatesSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertTrue(serviceComplTableSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertTrue(serviceSimplTableSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertTrue(serviceTableWithPagesSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertTrue(serviceDiffElemSubCategoryLeftPanel.isDisplayed());
+        softAssert.assertAll();
+
+        // 7. Open through the header menu Service -> Different Elements Page
+        serviceHeaderSubMenu.click();
+        serviceDiffElemSubCategory.click();
+
+        // 8. Check interface on Different elements page,
+        // it contains all needed elements
+        WebElement defaultButton = driver.findElement(By.name("Default Button"));
+        softAssert.assertTrue(defaultButton.isDisplayed());
+        WebElement Button = driver.findElement(By.cssSelector(".uui-button[type=button]"));
+        softAssert.assertTrue(Button.isDisplayed());
+        WebElement dropdown = driver.findElement(By.cssSelector(".colors"));
+        softAssert.assertTrue(dropdown.isDisplayed());
+        WebElement checkboxWater = driver.findElement(By.xpath("//label[@class='label-checkbox' and contains(.,'Water')]"));
+        softAssert.assertTrue(checkboxWater.isDisplayed());
+        WebElement checkboxEarth = driver.findElement(By.xpath("//label[@class='label-checkbox' and contains(.,'Earth')]"));
+        softAssert.assertTrue(checkboxEarth.isDisplayed());
+        WebElement checkboxWind = driver.findElement(By.xpath("//label[@class='label-checkbox' and contains(.,'Wind')]"));
+        softAssert.assertTrue(checkboxWind.isDisplayed());
+        WebElement checkboxFire = driver.findElement(By.xpath("//label[@class='label-checkbox' and contains(.,'Fire')]"));
+        softAssert.assertTrue(checkboxFire.isDisplayed());
```

Comment:
These elements could be get as `List<WebElement>`

Reply:
fixed